### PR TITLE
CHANGELOG: update change log for 0.2.2 release

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,15 @@
 # Changelog
+## [0.2.2] - 2019-02-03
+### New Features
+ - Team interface support
+
+### Bug Fixes
+ - Fix memory leak of NM.Client
+ - Get SR-IOV VFs configuration from runtime
+ - Fix DHCPv4 with static IPv6
+ - Use LC_TIME for RPM date
```
Comment (tyll) [inline CHANGELOG]: This change does not matter for users.

Comment (ffmancera) [reply CHANGELOG]: Removed. Thanks!

File: CHANGELOG [hunk 2]
```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
```
Comment (tyll) [inline CHANGELOG]: It is 2020 already and it is also not 3 February anymore. 0.2.1 also used the wrong year, please update it as well. :-)

Comment (tyll) [inline CHANGELOG]: It should probably contain a "Breaking Change" note that NM 1.22 is now required.

File: CHANGELOG [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+### New Features
+ - Team interface support
+
+### Bug Fixes
+ - Fix memory leak of NM.Client
```
Comment (tyll) [inline CHANGELOG]: I think this was a file descriptor leak, how about `Fix file descriptor leak` - the NM.Client part is an internal detail

File: CHANGELOG [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+ - Get SR-IOV VFs configuration from runtime
```
Comment (tyll) [inline CHANGELOG]: `from kernel instead of reading the NetworkManager configuration`

File: CHANGELOG [hunk 5]
```diff
@@ -1,4 +1,14 @@
...
+ - Fix DHCPv4 with static IPv6
+ - Learn the OVS Bridge ports and interfaces from NetworkManager
```
Comment (tyll) [inline CHANGELOG]: I am not sure what this really means and I guess users will not know, too.

File: CHANGELOG [hunk 6]
```diff
@@ -1,4 +1,14 @@
...
+
```
Comment (tyll) [inline CHANGELOG]: Missing is something like:
- Wait a short time before verification to allow the network state to stabilize

File: CHANGELOG (hunk 5 above)
Comment (ffmancera) [reply CHANGELOG]: Maybe that is something not important to users, it just means that now we are using libnm functions to get all bridge ports..

File: CHANGELOG (hunk 2 above)
Comment (ffmancera) [reply CHANGELOG]: Ooops, sorry. It seems my mind is not in 2020 yet :-)

Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

File: CHANGELOG (hunk 3 above)
Comment (ffmancera) [reply CHANGELOG]: Sure, done thanks!

File: CHANGELOG (hunk 4 above)
Comment (ffmancera) [reply CHANGELOG]: Done, thanks!

File: CHANGELOG (hunk 6 above)
Comment (ffmancera) [reply CHANGELOG]: Added. Thank you for point it out :)

**Pull request merged**